adminSDK/directory/index.html

```diff
@@ -22,128 +22,88 @@
+            gapi.auth2.getAuthInstance().isSignedIn.listen(updateSigninStatus);
+
+            // Handle the initial sign-in state.
+            updateSigninStatus(gapi.auth2.getAuthInstance().isSignedIn.get());
+            authorizeButton.onclick = () => gapi.auth2.getAuthInstance().signIn();
+            signoutButton.onclick = () => gapi.auth2.getAuthInstance().signOut();
+          });
         });
       }
 
       /**
-       *  Called when the signed in status changes, to update the UI
-       *  appropriately. After a sign-in, the API is called.
+       * Called when the signed in status changes, to update the UI
+       * appropriately. After a sign-in, the API is called.
        */
       function updateSigninStatus(isSignedIn) {
-        if (isSignedIn) {
-          authorizeButton.style.display = 'none';
-          signoutButton.style.display = 'block';
-          listUsers();
-        } else {
-          authorizeButton.style.display = 'block';
-          signoutButton.style.display = 'none';
-        }
+        if (isSignedIn) listUsers();
+        authorizeButton.style.display = (isSignedIn ? 'none' : 'block');
+        signoutButton.style.display = (isSignedIn ? 'block' : 'none');
       }
 
       /**
-       *  Sign in the user upon button click.
-       */
-      function handleAuthClick(event) {
-        gapi.auth2.getAuthInstance().signIn();
-      }
-
-      /**
-       *  Sign out the user upon button click.
-       */
-      function handleSignoutClick(event) {
-        gapi.auth2.getAuthInstance().signOut();
-      }
-
-      /**
-       * Append a pre element to the body containing the given message
-       * as its text node. Used to display the results of the API call.
-       *
+       * Append a pre element to the body with a message.
        * @param {string} message Text to be placed in pre element.
        */
       function appendPre(message) {
-        var pre = document.getElementById('content');
-        var textContent = document.createTextNode(message + '\n');
-        pre.appendChild(textContent);
+        document.getElementById('content')
+          .appendChild(document.createTextNode(message + '\n'));
       }
 
       /**
        * Print the first 10 users in the domain.
        */
       function listUsers() {
         gapi.client.directory.users.list({
-          'customer': 'my_customer',
-          'maxResults': 10,
-          'orderBy': 'email'
-        }).then(function(response) {
-          var users = response.result.users;
+          customer: 'my_customer',
+          maxResults: 10,
+          orderBy: 'email'
+        }).then((response) => {
+          const users = response.result.users;
           appendPre('Users:');
-
           if (users && users.length > 0) {
-            for (i = 0; i < users.length; i++) {
-              var user = users[i];
-              appendPre('-' + user.primaryEmail + ' (' + user.name.fullName + ')');
-            }
+            users.map((user, i) => {
```

Comment:
I prefer `forEach` over `map` when you don't intend to return a value. Sorry I didn't catch this in the first review.